examples/CMakeLists.txt

```diff
@@ -31,8 +31,14 @@ if(NOT WIN32)
 	target_link_libraries(cpp-multithread pthread)
 endif()
 
+_add_example(timer
```

Comment:
I wonder if the timer example should be merged with the cpp-multithread example...

Reply:
Single program with a button to choose whether to use a thread or `uiTimer()`.